runtime/services/catalog/migrator/metricsviews/metrics_views.go

```diff
@@ -75,8 +75,20 @@ func (m *metricsViewMigrator) Validate(ctx context.Context, olap drivers.OLAPSto
 
 	var validationErrors []*runtimev1.ReconcileError
 
+	columnNames := make(map[string]bool)
 	dimensionNames := make(map[string]bool)
 	for i, dimension := range mv.Dimensions {
+		if _, ok := columnNames[strings.ToLower(dimension.Column)]; ok {
```

Comment:
Is this needed because of backwards compatibility or do we not allow for duplicate columns anymore?

Reply:
Ah the API response is still returning column names instead of dimension name. We should perhaps change that?

Reply:
Yeah, we should definitely change that at some point :)